host.py

```diff
@@ -122,7 +123,47 @@ def _read_output(self, cmd: str, log_level: int = logging.DEBUG) -> Result:
 
         return Result(out, err, exit_code)
 
+    def remove(self, source):
+        if self.is_localhost():
+            if os.path.exists(source):
+                os.remove(source)
+        else:
+            try:
+                sftp = self._host.open_sftp()
+                sftp.remove(source)
+            except FileNotFoundError:
+                pass
+
+    def put_sftp(self, local_file, remote_file):
+        assert not self.is_localhost()
```

Comment:
Can we have it be more generic in the name? copy(..,...). I would like to hide the fact that it's a copy to remote as much as possible.